### ISSUE TITLE
Enable file downloads on WordPress

### ISSUE THREAD

**Issue by metmarkosaric** [plausible/wordpress#63]
We're about to release [automated file downloads](https://plausible.io/docs/file-downloads-tracking) this week so would be nice to enable this by default in the plugin similar to how we have external link clicks and 404 error pages enabled.

**Comment by masteradhoc** [comment]
+1 for this one :)

**Comment by metmarkosaric** [comment]
thanks @jdevalk!

**Comment by metmarkosaric** [comment]
New WordPress plugin release is now out! See https://wordpress.org/plugins/plausible-analytics/

File downloads are now included alongside external link click and 404 error page tracking

**Comment by jdevalk** [comment]
I see props in the changelog, but apparently my work was not prop worthy? :)

**Comment by Dan0sz** [comment]
Oops! You're totally right, @jdevalk! I'll add them now and commit it, so they'll appear in the next release (which should appear in a month or so)

I took over the development for this plugin at the beginning of last week, and since the plugin hasn't been updated in 2 years, I saw so many PRs and commits in the past week, I had no idea who to give props to anymore! ("Ik zag door de bomen het bos niet meer, zeg maar ;-))

Thanks for your efforts, @jdevalk!
